### PR TITLE
Moved advancedSearch module from pages to apps

### PR DESCRIPTION
**Pull request by mdavis-enquizit**
## Description
Patient Profile moved from pages to apps module

## Tickets

* [CNFT1-1887](https://cdc-nbs.atlassian.net/browse/CNFT1-1887)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ ] All new functions/classes/components reasonably small
- [ ] Functions/classes/components focused on one responsibility
- [ ] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ ] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

[CNFT1-1887]: https://cdc-nbs.atlassian.net/browse/CNFT1-1887?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GangulaMounika — APPROVED** [review]

File: apps/modernization-ui/src/apps/search/advancedSearch/components/chips/LabReportChips.tsx
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/search/advancedSearch/components/chips/LabReportChips.tsx:1]: This should be under the `apps/search/event/labReportSearch` folder.

File: apps/modernization-ui/src/apps/search/advancedSearch/components/chips/LabReportChips.spec.tsx
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/search/advancedSearch/components/chips/LabReportChips.spec.tsx:1]: This should be under the `apps/search/event/labReportSearch` folder.

File: apps/modernization-ui/src/apps/search/advancedSearch/components/LabReportResults.tsx
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/search/advancedSearch/components/LabReportResults.tsx:1]: This should be under the `apps/search/event/labReportSearch` folder.

File: apps/modernization-ui/src/apps/search/advancedSearch/components/LabReportResults.spec.tsx
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/search/advancedSearch/components/LabReportResults.spec.tsx:1]: This should be under the `apps/search/event/labReportSearch` folder.

**Comment by mdavis-enquizit** [comment]
Close for PR https://github.com/CDCgov/NEDSS-Modernization/pull/879/files

**Pull request closed without merging**